modules/install/pages/upgrade-to-ipv6.adoc

```diff
@@ -0,0 +1,42 @@
+= Upgrading to IPv6
+
+[abstract]
+A cluster that is upgraded to the latest version of Couchbase Server can be specified to use the IPv6 address family.
```

Comment:
We don't currently support changing address family during upgrade.

Sorry for confusion but the main purpose of this page is the following:
If a customer uses IPv6 cluster of one of following versions 5.5.0, 5.5.1, 5.5.2, or 6.0.0 they need to perform extra steps during offline upgrade (and only offline upgrade, online upgrade should work fine).

If they don't use IPv6 or uses other versions of CB or they want to perform online upgrade (instead of offline), they don't need to do anything unusual.

That's the only main idea. As a hint I think we can add a recommendation: if you plan to change address family, it is probably a good idea to upgrade to 6.5 first and to change address family after the upgrade, in order to avoid doing those extra steps.

Reply:
Ok. Take a look if you could at the summary I've provided under "IPv6 Cluster-Upgrade to 6.5". I hope I've gotten this right!